src/Symfony/Component/Intl/NumberFormatter/NumberFormatter.php

```diff
@@ -710,6 +710,7 @@ private function round($value, $precision)
         } elseif (isset(self::$customRoundingList[$roundingModeAttribute])) {
             $roundingCoef = pow(10, $precision);
             $value *= $roundingCoef;
+            $value = (float) (string) $value;
```

Comment:
why first casting to string and then to float again ? The `*=` operator means that this is probably not a string already. What happens if you remove this ?